src/www/widgets/widgets/interface_statistics.widget.php

```diff
@@ -118,26 +108,18 @@ function interface_statistics_widget_update(sender, data)
     <th>&nbsp;</th>
 <?php
     foreach ($interfaces as $ifdescr => $ifname):
-      $listed = in_array($ifdescr, $pconfig['interfacesstatisticsfilter']);
-      $listed = !empty($pconfig['interfacesstatisticsinvert']) ? $listed : !$listed;
-      if (!$listed) {
-        continue;
-      } ?>
+    if (!count($pconfig['interfacesstatisticsfilter']) || in_array($ifdescr, $pconfig['interfacesstatisticsfilter'])):?>
     <th id="interface_statistics_widget_intf_<?= html_safe($ifdescr) ?>"><?= $ifname ?></th>
-<?php endforeach ?>
+<?php endif; endforeach ?>
   </tr>
 <?php
     foreach ($ifvalues as $ifkey => $iflabel): ?>
   <tr id="interface_statistics_widget_val_<?= html_safe($ifkey) ?>">
     <td><strong><?= $iflabel ?></strong></td>
 <?php foreach ($interfaces as $ifdescr => $ifname):
-      $listed = in_array($ifdescr, $pconfig['interfacesstatisticsfilter']);
-      $listed = !empty($pconfig['interfacesstatisticsinvert']) ? $listed : !$listed;
-      if (!$listed) {
-        continue;
-      } ?>
+    if (!count($pconfig['interfacesstatisticsfilter']) || in_array($ifdescr, $pconfig['interfacesstatisticsfilter'])):?>
     <td>&#126;</td>
```

Comment:
if we are here "~" is an allowed character, no need to use an entity here.

Reply:
@fabianfrz does this work ? i hope this works!. would be great if you could help me get rid of these 3 files in this pr!